app/mlr/src/mlr_engine.cpp

```diff
@@ -400,7 +410,14 @@ std::string MLREngine::PrintAllEval(int32_t up_to_ith_eval) {
             loss_row[kColIdxLossTableNumEvalTest]);
       std::string num_test_str =
         std::to_string(static_cast<int>(loss_row[kColIdxLossTableNumEvalTest]));
-      test_info += test_zero_one_str + " " + num_test_str;
+      test_info += test_zero_one_str + " ";
+      if (perform_test_acc_) {
+        std::string test_acc_str = std::to_string(1 -
+          loss_row[kColIdxLossTableTestZeroOneLoss] /
```

Comment:
+1 you figure out how to use this loss_row structure! I appreciate the hacker style. However, Could you implement the code using interface like zero-one loss is implemented? basically it should have its own computation. Hopefully we could just replace zero-one loss once and for all.